examples/reminders/lib/base/di/app_dependencies.dart

```diff
@@ -6,8 +6,9 @@
 // https://opensource.org/licenses/MIT.
 
 import 'package:dio/dio.dart';
-import 'package:firebase_analytics/firebase_analytics.dart';
-import 'package:firebase_messaging/firebase_messaging.dart';
+// import 'package:firebase_analytics/firebase_analytics.dart';
```

Comment:
Same as below

Reply:
The comment lines were uncommented and a check `config != EnvironmentConfig.dev` was added